app/controllers/participate/login.js

```diff
@@ -15,7 +15,10 @@ export default Ember.Controller.extend({
     authenticate(attrs) {
       this.get('session')
         .authenticate('authenticator:jam-jwt', attrs)
-        .then(() => this.transitionToRoute('participate.survey.consent'))
+        .then(() => {
+          this.set('studyId', attrs['password']);
```

Comment:
It's being used here: https://github.com/CenterForOpenScience/isp/pull/53/files#diff-a022071735c5de65623e8b88c29ec2c6R6 -- let me know if you have another idea for how to get the studyId in the consent controller!